app/code/spec/Inviqa/SymfonyContainer/Helper/ContainerProviderSpec.php

```diff
@@ -10,7 +11,7 @@
 
 class Inviqa_SymfonyContainer_Helper_ContainerProviderSpec extends ObjectBehavior
 {
-    function let(Configuration $generatorConfig)
+    function let(Configuration $generatorConfig, Inviqa_SymfonyContainer_Model_StoreConfigCompilerPass $configCompilerPass)
```

Comment:
`Inviqa_SymfonyContainer_Model_StoreConfigCompilerPass` could be referenced by a shorter alias, so that the code is more readable:
```php
use Inviqa_SymfonyContainer_Model_StoreConfigCompilerPass as StoreConfigCompilerPass;
```